src/components/Select/AutosuggestMulti/AutosuggestMulti.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { bem } from '../../../utils';
+import { SuggestionTag } from './SuggestionTag';
+import { SuggestionsList } from '../SuggestionsList';
+import {
+    SelectBase,
+    CommonProps,
+    FocusedRendererHelpers,
+    BlurredRendererHelpers,
+} from '../SelectBase';
+import styles from './AutosuggestMulti.scss';
+import { BACKSPACE_KEY, ESCAPE_KEY, TAB_KEY } from '../../../constants';
+
+interface Props<S> extends CommonProps<S> {
```

Comment:
shouldn't be able to have a `clearButton` as well? In the example implementation I cannot delete all selected items at the moment.

Reply:
in current UI we don't have `clearButton`  and it is not clear where can place it.

Reply:
In the current Autosuggest it shows up if you hover over it when it is closed. I think that should be ok here too, unless @carlobernardini  thinks otherwise. It is actually already implemented only have to add this prop to pass the text for the button.

Reply:
<img width="525" alt="Screenshot 2020-03-26 at 18 18 27" src="https://user-images.githubusercontent.com/19436304/77676697-f6826000-6f8e-11ea-940f-d12efa3c1a4e.png">
Clear button moves tags to the left. We need to think up where to place it, because it can overlap tags.

Reply:
@carlobernardini can you advise?

Reply:
See Slack: if we can implement it without a lot of effort and make sure that the button does not flow out of the input, then I suggest we do it. The tags can be pushed to the left a little when the button appears on mouseover (possibly with a little transition?).